playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -38,3 +38,11 @@ network_config:
   - type: interface
     name: dummy1
     nm_controlled: true
+- type: sriov_pf
+  name: {{ sriov_interface }}
+  numvfs: 10
```

Comment:
numvfs should be a variable in defaults.yaml

Reply:
so I think we should use what we did in https://github.com/shiftstack/dev-install/pull/32/commits/6a33b009b400508fa58f827ed892acd490178f6a to override the network_config when needed.
Or simply do what I think @mdbooth would prefer:

```
{% if sriov_interface is defined %}
- type: sriov_pf
  name: {{ sriov_interface }}
  numvfs: {{ sriov_nic_numvfs | defauly(10) }}
  use_dhcp: false
  defroute: false
  nm_controlled: true
  hotplug: true
  promisc: false
{% endif %}
```

I have no strong opinion, @adduarte if you think this config ^ can work in a majority of SRIOV configs, then I think this is good for dev-install.